test/gateway/ost_prime/unwrap.js

```diff
@@ -0,0 +1,192 @@
+const OSTPrime = artifacts.require("TestOSTPrime")
+  , BN = require('bn.js');
+
+const Utils = require('../../../test/test_lib/utils');
+const EventDecoder = require('../../test_lib/event_decoder.js');
+
+contract('OSTPrime.sol', function (accounts) {
+
+  describe('Unwrap', async () => {
+
+    const DECIMAL = new BN(10);
+    const POW = new BN(18);
+    const DECIMAL_FACTOR = DECIMAL.pow(POW);
+    const TOKENS_MAX = new BN(800000000).mul(DECIMAL_FACTOR);
+
+    let brandedTokenAddress, ostPrime, callerAddress, amount;
+
+    async function initialize(){
+      await ostPrime.initialize(
+        {from: accounts[2], value:TOKENS_MAX}
+      );
+    };
+
+    beforeEach(async function () {
+
+      brandedTokenAddress = accounts[2];
+      ostPrime = await OSTPrime.new(brandedTokenAddress);
+
+      callerAddress = accounts[3];
+      amount = new BN(500);
+
+      await ostPrime.setTokenBalance(callerAddress, amount);
+
+    });
+
+    it('Should fail when the amount is zero', async function () {
```

Comment:
We start all `it()`s with lowercase (see also other function calls)
```suggestion
    it('should fail when the amount is zero', async function () {
```